libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/speedlimit/SpeedLimitBitmapProvider.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.androidauto.navigation.speedlimit
+
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import com.bumptech.glide.load.engine.bitmap_recycle.LruBitmapPool
+import com.mapbox.navigation.base.speed.model.SpeedLimitSign
+
+internal class SpeedLimitBitmapProvider {
```

Comment:
```suggestion
class SpeedLimitBitmapRenderer {
```

Will you rename to be more consistent with the other renderers? "providers" are usually constructing data objects.

 `CarLanesImageRenderer`, `CarRoadLabelBitmapRenderer`, `CarSpeedLimitRenderer`

Reply:
Done